packages/shared/src/graphql/posts.ts

```diff
@@ -103,6 +104,7 @@ export const POST_BY_ID_QUERY = gql`
         username
       }
       description
+      summary
```

Comment:
We need to add this to `POST_BY_ID_STATIC_FIELDS_QUERY` so it will be cached for the next visits.
In the article page itself you can find this function `getStaticProps` that fetches the necessary data for the page before it arrives to the client and then we revalidate on the client side as well. I highly suggest reading about nextjs's incremental static regeneration for better understanding what's going on under the hood.

@rebelchris @sshanzel  this can be another valid point for future reviews in pages that use ISR or SSR.

Reply:
On this topic, I get the difference, but why can't they use the same GraphQL endpoint?

Reply:
Because some of the properties depend on the logged-in user, this will not be available and might even throw an exception. We can try that of course.